### PR TITLE
Modularized graphql schema.

### PR DESCRIPTION
**Pull request by benwarfield-usds**
Moved graphql schema from one mammoth file into three smaller and (somewhat) logically separated files.

### CODE REVIEW

File: backend/src/main/resources/graphql/main.graphqls [hunk 1]
```diff
@@ -198,74 +168,48 @@ type TestResult {
...
+
 type Query {
+  organization: Organization
   deviceType: [DeviceType]
   patients(
     facilityId: ID
     pageNumber: Int
     pageSize: Int
-    showDeleted: Boolean = false @requiredPermissions(allOf: ["READ_ARCHIVED_PATIENT_LIST"])
+    showDeleted: Boolean = false
```
Comment (benwarfield-usds) [inline backend/src/main/resources/graphql/main.graphqls:187]: There's a bunch of VSCode/prettier rearrangement here. My normal rule is to not do those rearrangements, but this is actually a rearrange-and-clean-up story, so I'm less opposed to it than usual. OTOH, if people dislike this rearrangement, I should back it out and fix my VSCode configuration. 😁

Comment (benwarfield-usds) [reply backend/src/main/resources/graphql/main.graphqls:187]: Tagging @jeskew-gov for directive-managing opinions.

Comment (jeskew-gov) [reply backend/src/main/resources/graphql/main.graphqls:187]: Moving directives to a new line looks just as readable to me. It's best to do this now while the file is being split to minimize noise in the git log for any given line.

File: backend/src/main/resources/graphql/admin.graphqls [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  clearQueue: Int
+  addUser(
+    firstName: String
+    middleName: String
+    lastName: String!
+    suffix: String
+    email: String!
+    organizationExternalId: String!
+    role: Role!
+  ): User @requiredPermissions(allOf: ["MANAGE_USERS"])
```
Comment (jeskew-gov) [inline backend/src/main/resources/graphql/admin.graphqls:30]: Could you remove this permission? This should site admins only, but I set it to the same level as `addUserToCurrentOrg` by mistake

**Review by jeskew-gov — APPROVED** [review]

**Pull request merged**